### PR TITLE
16.0 misc: grammar updates remove 2nd person nimk

### PR DESCRIPTION
**Pull request by MaxKeilOdoo**
Updated grammar per [[In-App Purchase (IAP)] Update ticket](https://www.odoo.com/web#id=3453291&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form) from DADE.

-Removed inactive hyperlink to “Buying Credits” section
-Updated grammar throughout: removed 2nd person
-Added and adjusted language to improve clarity throughout
-Changed titles to be sentence case
-Added alt text for all images
-Changed navigation instructions to use search function for ease of use
-Replaced first image to include Search bar per the updated instructions
-Adjusted formatting throughout for things like :guilabel:

### CODE REVIEW

File: content/applications/general/in_app_purchase.rst [hunk 1]
```diff
@@ -2,58 +2,63 @@
...
+.. _Buy credits:
 
-Buying Credits
+Buying credits
 ==============
 
-Each IAP Service relies on prepaid credits to work and has its own pricing. To consult your current
-balance or to recharge your account, go to :menuselection:`Settings --> Odoo IAP --> View my
-Services`.
+Each IAP service relies on prepaid credits to work and has its own pricing. To check an account’s
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:13]:
```suggestion
Each IAP service relies on prepaid credits to work and has its own pricing. To check an account's
```

File: content/applications/general/in_app_purchase.rst [hunk 2]
```diff
@@ -2,58 +2,63 @@
...
+current balance or to buy more credits (:guilabel:`Recharge`), go to the
+:menuselection:`Settings app --> Search "IAP" --> View My Services`.
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:15]:
```suggestion
current balance or to buy more credits (:guilabel:`Recharge`), go to the :menuselection:`Settings
app --> Search "IAP" --> View My Services`.
```

File: content/applications/general/in_app_purchase.rst [hunk 3]
```diff
@@ -2,58 +2,63 @@
...
 
-Credits to use IAP services are stored on IAP accounts, which are specific to each service.
-By default, IAP accounts are common to all companies, but can be restricted to specific
-ones. Activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Technical
-Settings --> IAP Account`.
+Credits for IAP services are stored on IAP accounts, which are specific to each service. By default,
+IAP accounts are common to all companies, but can be restricted to specific ones. To make changes to
+IAP accounts, activate Odoo’s :ref:`developer mode <developer-mode>`. Return to the main apps page
+and type *IAP*. In the resulting dropdown menu select
+:guilabel:`Settings / Technical / IAP / IAP Accounts`.
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:31]:
```suggestion
IAP accounts, activate Odoo's :ref:`developer mode <developer-mode>`. Return to the main apps page
and type *IAP*. In the resulting drop down menu select :guilabel:`Settings / Technical / IAP / IAP
Accounts`.
```

File: content/applications/general/in_app_purchase.rst [hunk 4]
```diff
@@ -2,58 +2,63 @@
...
-IAP Portal
+IAP portal
 ==========
 
-The IAP Portal is a platform regrouping your IAP Services. It is accessible from
-:menuselection:`Settings app --> Odoo IAP --> View my Services`. From there, you can view your current
-balance, recharge your credits and set a reminder when your balance falls below a threshold.
+The IAP portal is a platform for managing all IAP services. Actions that can be performed in the
+portal include checking an account's balance, buying more credits (recharge), and setting reminders
+to be automatically notified when an account’s balance falls below a specified threshold. To access,
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:46]:
```suggestion
to be automatically notified when an account's balance falls below a specified threshold. To access,
```

File: content/applications/general/in_app_purchase.rst [hunk 5]
```diff
@@ -2,58 +2,63 @@
...
-To be notified when it’s time to recharge your credits, you can go to your IAP Portal through
-:menuselection:`Settings app --> Odoo IAP --> View my Services`, unfold a service and check the
-Receive threshold warning option. Then, you can provide a minimum amount of credits and email
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+To be notified when credits are low, go to the
+:menuselection:`Settings app --> Search "IAP" --> View My Services`. Unfold a service and check the
+:guilabel:`Receive threshold warning` box. Enter a minimum credit amount in the
+:guilabel:`Warning threshold` field. Finally, click :guilabel:`Change Email` and provide an email
+address to be automatically notified when credits are at or below the :guilabel:`Warning threshold`.
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:60]:
```suggestion
To be notified when credits are low, go to the :menuselection:`Settings app --> Search "IAP" -->
View My Services`. Unfold a service and check the :guilabel:`Receive threshold warning` box. Enter a
minimum credit amount in the :guilabel:`Warning threshold` field. Finally, click :guilabel:`Change
Email` and provide an email address to be automatically notified when credits are at or below the :guilabel:`Warning threshold`.
```

File: content/applications/general/in_app_purchase.rst [hunk 6]
```diff
@@ -2,58 +2,63 @@
...
 
 .. image:: in_app_purchase/image4.png
    :align: center
+   :alt: The Receive threshold warning box is checked and a Warning threshold is set.
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:64]: May want to add a section on manually editing the token in Odoo and where to edit (dev mode --> settings app--> technical menu-->
IAP-->
IAP Accounts
OR
IAP Partners

File: content/applications/general/in_app_purchase.rst [hunk 7]
```diff
@@ -2,58 +2,63 @@
...
 
 .. image:: in_app_purchase/image2.png
    :align: center
+   :alt: The IAP Account menu showing IAP services, companies, and account tokens.
 
 .. tip::
-   An IAP account can be disabled by appending `+disabled` to its token.
-   Reverting this change will re-enable the account.
+   To disable an IAP account, append `+disabled` to its token. To re-enable the account, remove
+   `+disabled` from the token.
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:39]: where? technical menu IAP-> IAP accounts (in dev mode)

File: content/applications/general/in_app_purchase.rst [hunk 8]
```diff
@@ -2,58 +2,63 @@
...
-:menuselection:`Settings app --> Odoo IAP --> View my Services`. From there, you can view your current
-balance, recharge your credits and set a reminder when your balance falls below a threshold.
+The IAP portal is a platform for managing all IAP services. Actions that can be performed in the
+portal include checking an account's balance, buying more credits (recharge), and setting reminders
+to be automatically notified when an account’s balance falls below a specified threshold. To access,
+the IAP portal, go to the :menuselection:`Settings app --> Search "IAP" --> View My Services`.
 
 .. image:: in_app_purchase/image3.png
    :align: center
+   :alt: The Your Services page showing selected services, current balance, and the Recharge button.
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:51]:
```suggestion
   :alt: The My Services page showing selected services, current balance, and the Recharge button.
```

**Review by tiku-odoo — COMMENTED** [review]
@MaxKeilOdoo 

Some edits for the IAP doc. Nice work. Let me know if you have any questions. 
I will approve once you make the edits....

Tag: JERO or LARM next! 

Thanks,
Tim

File: content/applications/general/in_app_purchase.rst (hunk 7 above)
Comment (MaxKeilOdoo) [reply content/applications/general/in_app_purchase.rst:39]: Expanded this tip to be part of the overall IAP accounts section, added content for making changes and fuller steps for disabling an account. Also added a new image showing "+disabled" at the end of an account token.

File: content/applications/general/in_app_purchase.rst (hunk 6 above)
Comment (MaxKeilOdoo) [reply content/applications/general/in_app_purchase.rst:63]: agreed, added this to the IAP Accounts section

File: content/applications/general/in_app_purchase.rst [hunk 9]
```diff
@@ -2,58 +2,70 @@
...
+
+.. _Buy credits:
+
+Buying credits
+==============
+
+Each IAP service relies on prepaid credits to work and has its own pricing. To check an account’s
+current balance or to buy more credits (:guilabel:`Recharge`), go to the
+:menuselection:`Settings app --> Search "IAP" --> View My Services`.
```
[the remark this answers is not in the document]
Comment (tiku-odoo) [reply content/applications/general/in_app_purchase.rst:51]: Make sure to fill out the 100 character line limit as best as possible

File: content/applications/general/in_app_purchase.rst [hunk 10]
```diff
@@ -2,58 +2,70 @@
...
+
+.. image:: in_app_purchase/search-for-iap-in-settings.png
+   :align: center
+   :alt: General Settings and the In-App Purchases heading showing the View My Services button.
+
+.. tip::
+   Users who have the Enterprise version of Odoo Online get free credits to test IAP features.
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:58]: Do you know the number of the basic free credits?

File: content/applications/general/in_app_purchase.rst [hunk 11]
```diff
@@ -2,58 +2,70 @@
...
-To be notified when it’s time to recharge your credits, you can go to your IAP Portal through
-:menuselection:`Settings app --> Odoo IAP --> View my Services`, unfold a service and check the
-Receive threshold warning option. Then, you can provide a minimum amount of credits and email
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+To be notified when credits are low, go to the
+:menuselection:`Settings app --> Search "IAP" --> View My Services`. Click on a service and check
+the :guilabel:`Receive threshold warning` box. Enter a minimum credit amount in the
+:guilabel:`Warning threshold` field. Finally, click :guilabel:`Change Email` and provide an email
+address to be automatically notified when credits are at or below the :guilabel:`Warning threshold`.
```
Comment (tiku-odoo) [inline content/applications/general/in_app_purchase.rst:67]:
```suggestion
To be notified when credits are low, go to the :menuselection:`Settings app --> Search "IAP" -->
View My Services`. Click on a service and check the :guilabel:`Receive threshold warning` box. Enter
a minimum credit amount in the :guilabel:`Warning threshold` field. Finally, click :guilabel:`Change
Email` and provide an email address to be automatically notified when credits are at or below the :guilabel:`Warning threshold`.
```

Comment (tiku-odoo) [reply content/applications/general/in_app_purchase.rst:67]: need to fill out the 100 character limit

**Review by tiku-odoo — COMMENTED** [review]
@MaxKeilOdoo 

You made some good edits, nice work..

Some additional comments. I would like to see you dive deeper into IAP and guide the user to things you find out. Doc could be fleshed out a bit more. Big thing is to access via the settings and be redirected with the logging to IAP.odoo.com. 

Another thing to keep in mind is the 100-character limit. You're able to break up the labels on different lines. Let me know when you make some more changes and I'll take another look. 

Thanks, Tim :+1:

**Comment by tiku-odoo** [comment]
@MaxKeilOdoo There are some build errors on this PR- it looks as if your on the wrong base branch somewhere along the line. I would touch base with @samueljlieber to correct these errors.  :+1:

**Comment by MaxKeilOdoo** [comment]
Hi @tiku-odoo , I just pushed a second draft of this and need your review. That said, I ran into a few issues and could not complete the full work flow or find descriptions of a few of the IAP services. Please take a look when you can, but I would also like to hop on a quick call to discuss quickly patching in the missing content.

Thank you, and please let me know if you have any further questions or suggestions.

**Comment by StraubCreative** [comment]
Hi @MaxKeilOdoo 

Based on the [CI check errors](https://runbot.odoo.com/runbot/build/56564126), I think we'll need to cherry-pick your work into a new PR here.

Runbot is building your feature branch on `master` which is the incorrect base branch, and is therefore causing all of the CI checks to fail. The reason why this is happening, is likely due to the branch not following naming convention (e.g. `16.00` vs. `16.0`) which informs runbot on how to build instances correctly.

**Comment by MaxKeilOdoo** [comment]
ATTENTION: I am closing this pr and continuing work in [#7427](https://github.com/odoo/documentation/pull/7427)

**Pull request closed without merging**